src/lua/bpf/LICENSE

```diff
@@ -0,0 +1,23 @@
+Copyright (c) 2016, Marek Vavrusa <marek@vavrusa.com>
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
```

Comment:
do you have a strong reason to choose bsd 2-clause license?
the rest of the bcc is covered by apache2 license.
the main reason we picked apache2 is patent rights.